latest/wfi_mosaic.yaml

```
@@ -13,49 +13,324 @@ type: object
+                description: >
+                  Pixel scale in units of arcseconds per pixel at the
+                  center of the projection region. If this product does not use
+                  the Roman sky tessellation, then this keyword will be equal to
+                  meta.wcsinfo.pixel_scale.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.pixel_scale_ref]
+              pixel_scale:
+                title: Pixel Scale at Image Center (arcseconds per pixel)
+                description: >
+                  Pixel scale in units of arcseconds per pixel at the center of the image in this file.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.pixel_scale]
+              projection:
+                title: Projection Type
+                description: >
+                  Type of projection used for the world coordinate
+                  system (WCS). A value of "TAN" indicates a tangent plane projection.
+                type: string
+                enum: ["TAN"]
+                maxLength: 50
+                archive_catalog:
+                  datatype: nvarchar(50)
+                  destination: [WFIMosaic.projection]
+              s_region:
+                title: Spatial Extent of the Image
+                description: >
+                  The region of the sky enclosed by the pixel data
+                  contained within this file. This is given as a polygon stored as
+                  a string with the vertices defined by a list of right ascension
+                  and declination pairs in units of degrees.
+                type: string
+                archive_catalog:
+                  datatype: nvarchar(max)
+                  destination: [WFIMosaic.s_region]
+              image_shape:
+                title: Shape of Image (pixels)
+                description: Shape of the image (nx, ny) in units of pixels.
+                type: array
+                minItems: 2
+                maxItems: 2
+                items:
+                  type: integer
+                archive_catalog:
+                  datatype: nvarchar(50)
+                  destination: [WFIMosaic.image_shape]
+              ra:
+                title: Right Ascension of Image Center (deg)
+                description: Right ascension in units of degrees at the center of the image.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.ra]
+              dec:
+                title: Declination of Image Center (deg)
+                description: Declination in units of degrees at the center of the image.
+                type: number
+                archive_catalog:
+                  datatype: float
+                  destination: [WFIMosaic.dec_center]
```

This is still dec_center?

```suggestion
                  destination: [WFIMosaic.dec]
```

---

Thanks! I fixed this in: 5c34527